roles/container-engine/containerd/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+containerd_version: 1.2.6
```

Comment:
version and download url's must be moved to the downloads role

Reply:
Use containerd.io package included in docker-ce. Now don't need define containerd.